modules/fullwidth.js

```diff
@@ -7,7 +7,17 @@ module.exports = {
         let codePoints = Array.prototype.map.call(msg.body, (c) => c.codePointAt(0))
         codePoints = codePoints.map((c) => {
           if (c >= 33 && c <= 126) return c - 33 + 0xFF01
-          if (c === 32) return 0x3000
+          let nonAscii = {
```

Comment:
as this never changes, you might want to put the immutability in writing by using `const` instead of `let`